src/shared/db.ts

```diff
@@ -580,4 +585,51 @@ export class DB {
 
         return hash
     }
+
+    // PluginLogEntry
+
+    public async fetchPluginLogEntries(): Promise<PluginLogEntry[]> {
+        if (this.kafkaProducer) {
+            return (await this.clickhouseQuery(`SELECT * FROM plugin_log_entries`)).data as PluginLogEntry[]
+        } else {
+            return (await this.postgresQuery('SELECT * FROM posthog_pluginlogentry')).rows as PluginLogEntry[]
+        }
+    }
+
+    public async createPluginLogEntry(
+        pluginConfig: PluginConfig,
+        type: PluginLogEntryType,
+        message: string,
+        instanceId: UUID
+    ): Promise<PluginLogEntry> {
+        const entry: PluginLogEntry = {
+            id: new UUIDT().toString(),
+            team_id: pluginConfig.team_id,
+            plugin_id: pluginConfig.plugin_id,
+            timestamp: new Date().toISOString().replace('T', ' ').replace('Z', ''),
+            type,
+            message,
+            instance_id: instanceId.toString(),
```

Comment:
Thinking out loud: could it make sense to store `pluginConfig` id as well in the tables? Also below in `pluginDigest`. Filtering by `pluginconfig_id` feels a bit safer somehow: no risk of forgetting `team_id` and showing the wrong things.

Reply:
Why not, makes sense